models/marts/daysheet/mrt_daysheet_cashpayments.sql

```diff
@@ -0,0 +1,66 @@
+{{ config(
+    materialized = 'table',
+    schema="marts",
+    sort = ['doctor_id', 'posted_date', 'scheduled_time']
+) }}
+
+
+SELECT
+    -- CashPayment
+    bcp.*,
+    -- Patient
+    cp.first_name,
+    cp.middle_name,
+    cp.last_name,
+    cp.chart_id,
+    co.name,
+    -- Doctor
+    cd.firstname,
+    cd.lastname,
+    cd.salutation,
+    cd.suffix,
+    cd.practice_group_id,
```

Comment:
```suggestion
    cd.id as doctor_id, 
    cd.practice_group_id,
```
We need to include doctor_id for RLS as well

Reply:
Yep `doctor_id` is included from the CashPayment table